extensions/azdata/src/test/common/childProcess.test.ts

```diff
@@ -9,21 +9,22 @@ import * as TypeMoq from 'typemoq';
 import { executeCommand } from '../../common/childProcess';
 
 describe('ChildProcess', function () {
-	[undefined, [], ['test']].forEach(args => {
+	const outputChannelMock = TypeMoq.Mock.ofType<vscode.OutputChannel>();
+
+	[[], ['test']].forEach(args => {
```

Comment:
nit suggestion: In general by default it is better to use 'for of' intead of forEach unless there is a specific reason to use forEach.
https://thecodebarbarian.com/for-vs-for-each-vs-for-in-vs-for-of-in-javascript

Reply:
I find it more readable and none of the issues described there are relevant to this usag.e